tests/test-war-smoke/pom.xml

```diff
@@ -228,35 +167,146 @@
+    <profile>
+      <id>test.remote.deploy</id>
+      <build>
+        <plugins>
+          <!-- create tag for local docker image to push for remote use in test -->
+          <plugin>
+            <groupId>com.spotify</groupId>
+            <artifactId>docker-maven-plugin</artifactId>
+            <executions>
+              <execution>
+                <id>tag-image</id>
+                <phase>package</phase>
+                <goals>
+                  <goal>tag</goal>
+                </goals>
+                <configuration>
+                  <image>jetty:${docker.tag.short}</image>
+                  <newName>${jetty.test.image}</newName>
+                </configuration>
+              </execution>
+            </executions>
+          </plugin>
+          <plugin>
+            <groupId>org.codehaus.mojo</groupId>
+            <artifactId>exec-maven-plugin</artifactId>
+            <executions>
+              <!-- push a remote docker image to use with 'deploy-integration-test' -->
+              <execution>
+                <id>deploy-remote-docker</id>
+                <phase>package</phase>
+                <goals>
+                  <goal>exec</goal>
+                </goals>
+                <configuration>
+                  <executable>gcloud</executable>
+                  <arguments>
+                    <argument>docker</argument>
+                    <argument>-q</argument>
+                    <argument>--</argument>
+                    <argument>push</argument>
+                    <argument>${jetty.test.image}</argument>
+                  </arguments>
+                </configuration>
+              </execution>
+            </executions>
+          </plugin>
+        </plugins>
+      </build>
+    </profile>
+    <profile>
+      <id>test.remote.clean</id>
+      <build>
+        <plugins>
+          <plugin>
+            <groupId>org.codehaus.mojo</groupId>
+            <artifactId>exec-maven-plugin</artifactId>
+            <executions>
+              <!-- remove the remote image from 'deploy-remote-docker' -->
+              <execution>
+                <id>cleanup-remote-image</id>
+                <phase>post-integration-test</phase>
+                <goals>
+                  <goal>exec</goal>
                 </goals>
                 <configuration>
-                  <deploy.project>${app.deploy.project}</deploy.project>
-                  <deploy.version>${maven.build.timestamp}</deploy.version>
-                  <deploy.promote>false</deploy.promote>
-                  <dockerDirectory>${project.build.directory}/docker
-                  </dockerDirectory>
+                  <executable>gcloud</executable>
+                  <arguments>
+                    <argument>beta</argument>
+                    <argument>container</argument>
+                    <argument>images</argument>
+                    <argument>delete</argument>
+                    <argument>${jetty.test.image}</argument>

```

Comment:
This whole command doesn't actually work, it turns out. We need to specify the sha256 digest of the image instead of the version tag. See: https://cloud.google.com/sdk/gcloud/reference/beta/container/images/delete
Do you think it's possible to capture the image digest in a maven property somehow? It would be very helpful.